### PR TITLE
Nurse AI now responds to noises (e.g. pill bottles, breaking plates)

### PR DESCRIPTION
**Pull request by fnnbrr**
I don't think the noise ping collisions are 100% reliable for some reason; I've had instances where my ping ring has hit a nurse, but they didn't move to investigate it.  Could also be an error with my AI update function logic, so any input would be welcome.

### CODE REVIEW

File: Assets/Scripts/Interactables/Interactable.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ protected void OnTriggerExit(Collider other)
     // Handle the dialogue for this interactable
     public virtual void OnInteract()
     {
-        Debug.Log("Interacted");
+        //Debug.Log("Interacted");
```
Comment (jurcaua) [inline Assets/Scripts/Interactables/Interactable.cs]: Might as well just remove this line I guess!

File: Assets/Scripts/LoudObjects/LoudObject.cs [hunk 2]
```diff
@@ -53,6 +54,7 @@ private void OnCollisionEnter(Collision other)
 
         noisePing.SpawnNoisePing(other);
         hasHit = true;
+        Destroy(gameObject, despawnSec);
```
Comment (jurcaua) [inline Assets/Scripts/LoudObjects/LoudObject.cs]: I can make an issue to connect the object fade out with this, maybe after alpha or if I have extra time before then :)

File: ProjectSettings/TagManager.asset [hunk 3]
```diff
@@ -4,7 +4,7 @@
 TagManager:
   serializedVersion: 2
   tags:
-  - Enemy
```
Comment (jurcaua) [inline ProjectSettings/TagManager.asset:7]: Didn't we need the Enemy tag?
`PlayerManager.cs`

image

**Review by jurcaua — COMMENTED** [review]
Just some initial comments just looking at the code! Gonna dig in and checkout the branch later and test it out myself 😋

**Comment by jurcaua** [comment]
After testing it around, looks like it's working great!! I think we should definitely have some visual indicators like you said. Maybe for now we can do the "player caught" flashing instead? That would be a good temp fix.

Also I would wayyyyy up the radius for the throwables. It was super hard to get it right for me, especially since they don't really hang out close enough to the pantry counter to get into the range of anything I threw over of it. I think having it too large is better than too small (especially since this is a still kind of a "tutorial level" I guess?

Also., idea for the ping:
Maybe we can spawn the ping on the nearest point on the ground, and orient it so it's always flat on the ground? 
Right now if we throw and it hits a wall or a fridge or something, it will rarely get a OnTriggerEnter I feel, rather than if everything was based on the ground (where the enemy is guaranteed to be touching). Thoughts?

**Comment by fnnbrr** [comment]
Ping spawning on the ground sounds like a good idea, I'll rig it up to do just that!

**Comment by fnnbrr** [comment]
I'll mess around with the noise radius until it seems fair & decently easy.
I'm planning to also change the Responders to follow patrol points instead, just so it's guaranteed that they'll pass in front of the pantry in a timely manner.

File: ProjectSettings/TagManager.asset (hunk 3 above)
Comment (fnnbrr) [reply ProjectSettings/TagManager.asset:7]: Yeah I need to add that back, good catch.

**Comment by fnnbrr** [comment]
Alright @jurcaua, the responders are set up to follow a preset patrol route now and the mission is definitely more fair & playable!  Just tried it out and narrowly made it through.

One important property we can modify to scale the difficulty is the number of "searches" that Patrollers will do after hearing noise before they return to their default patrol route.

**Comment by jurcaua** [comment]
Hey, just testing it out, I'm getting a crash, probably from an infinite loop somewhere, the moment I am caught by an enemy. 2/2 reproduction rate so far. Can you try it out too? @FinnbarrOC

**Comment by jurcaua** [comment]
The mission itself definitely looked better than before and I had fun playing through it (before getting caught and catching that issue).

**Review by jurcaua — APPROVED** [review]
Should be good to go now :P Thank you for looking into the issue for so long!!

**Pull request merged**